src/Proto.Actor/Scheduler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Proto
+{
+    public static class Scheduler
+    {
+        public static void ScheduleTellOnce(this IContext context, TimeSpan delay, PID target, object message)
+        {
+            Task.Run(async () =>
+            {
+                await Task.Delay(delay);
+
+                context.Tell(target, message);
```

Comment:
That would be safe, _but_ what is the semantics we want here?
Should scheduled messages use the outbound middleware?
e.g. if you have a tracing middleware, should the trace continue when scheduling?
(imo yes, and if so, we need to deal with this in this code)